docs/react-integration.asciidoc

```diff
@@ -66,6 +66,11 @@ class App extends React.Component {
 }
 ----
 
+NOTE: `ApmRoute` only instruments component property if provided, in other cases, e.g. using `render` or `children` properties, 
```

Comment:
From reading through the doc first, it was not really clear - Probably we can also add what do you mean by component property - https://reacttraining.com/react-router/web/api/Route/component

To rephrase, 
```
`ApmRoute` instruments the route change event only when the route contains a [component property](https://reacttraining.com/react-router/web/api/Route/component). In other cases eg: when using render or children propeties, `ApmRoute` will not instrument the route-change event. 

please instrument the individual component using `withTransaction` in these cases instead.
```

Reply:
I've rephrased it a little bit! But let's keep the external links to a minimum, since they might change. I think in this context it's clear what component property is referring to, given that there's an example above it as well.